op_builder/builder.py

```diff
@@ -651,9 +651,9 @@ def compute_capability_args(self, cross_compile_archs=None):
         args = []
         self.enable_bf16 = True
         for cc in ccs:
-            num = cc[0] + cc[2]
+            num = cc[0] + cc[1].split('+')[0]
```

Comment:
Could you please update the comment lines 615 & 616 with Blackwell compute? This will help understanding of this code. Thanks!

Reply:
@tjruwase, fair point. I added the Hopper and Blackwell compute capabilities in the comments at lines 615 & 616.